### ISSUE TITLE
How to collect Schema.org Dataset criteria

### ISSUE THREAD

**Issue by bleonar5** [psych-ds/psych-DS#28]
# TL;DR
We want to be able to exhaustively identify the set of Schema.org types and properties that could appear in the context of the Dataset type.

# Detailed View
It seems that there is no option available for an open source, programmatically accessible Schema.org validator. The code behind the freely available validator [here](https://validator.schema.org/) seems intentionally opaque and unavailable. In lieu of something like this, we'll need to combine freely available JSON-LD packages with static info about the Dataset schema in order to successfully validate for both JSON-LD and Schema.org/Dataset.

Is there an API from which we can pull Schema.org types and properties automatically? Is there some way of rigorously crawling the website to develop a taxonomy of types and properties? Is it possible to automatically update our internal validator schemas when a change is detected in the canonical Schema.org structure?

In addition to implementing what we need for the current schema, we also need to establish plans for versioning & handling future schema.org updates.
